src/main/java/io/github/protonmc/proton/module/decoration/tater/FatPotatoBlock.java

```diff
@@ -0,0 +1,33 @@
+package io.github.protonmc.proton.module.decoration.tater;
```

Comment:
How would @dzwdz 's category detection system handle this?

Reply:
Absolutely fine.

Reply:
There is no category named `TATER`

Reply:
The category is called decoration though..

Reply:
@TehcJS this needs to be `module.building` probably

Reply:
> The category is called decoration though..

I checked and the category detection checks for the last package part, which is `tater`.